wazo_confd/plugins/user/schema.py

```diff
@@ -50,6 +50,7 @@ class UserSchema(BaseSchema):
     lastname = fields.String(validate=Length(max=128), allow_none=True)
     email = fields.Email(validate=Length(max=254), allow_none=True)
     timezone = fields.String(validate=Length(max=128), allow_none=True)
+    country = fields.String(dump_only=True)
```

Comment:
Should update wazo_confd/plugins/user/api.yml with `country` as `readOnly: true` too.

Reply:
Done, thanks